More posters

Atakku opened the pull request:
<!-- Read upstream guidelines: https://docs.spacestation14.io/en/getting-started/pr-guideline -->
<!-- Not following the guidelines or removing any of these sections may result in your PR getting closed. -->

## General information
<!-- Describe your PR as you would to a regular player, omit technical details.
       Make sure to cover all player facing changes as well and as detailed
       as you can, as this will be forwarded to Discord later. -->
Added a bunch of nice posters from other downstreams

## Why / Balance
<!-- Discuss how this would affect game balance or explain why it was changed.
       Link any relevant discussions or issues. -->
More posters... more variety... much cool....

## Technical details
<!-- Describe what code changes you did or will do in this PR and optionally why.
       Feel free to use check boxes to track your progress, for example:-->
- [x] Yoinked a bazillion posters and fixed up their copyright notices.

## Media
<!-- Attach media if the PR makes in-game changes (clothing, items, features, etc).
       Small fixes/refactors are exempt. Media will be used when forwarded to Discord -->
<img width="631" height="647" alt="image" src="https://github.com/user-attachments/assets/71f3df14-18e0-4f70-ab02-e2fded6f02ec" />

## Breaking changes
<!-- List any breaking changes, including namespaces, public class/method/field changes,
       prototype renames, and provide instructions for fixing them. -->
None

## Changelog
<!-- Edit this to cover all player facing changes in a concise and short matter.
       Feel free to remove or add as many add, remove, fix, tweak lines as you need.
       If there are no player facing changes, you can remove it altogether.
       This will be visible in-game and on Discord. -->
:cl:
- add: Added 19 more posters! (15 regular and 4 contraband)

Atakku committed:
Groundwork for new posters

Atakku committed:
.rsi's

SnappingOpossum commented on Resources/Prototypes/_Carpmosia/Entities/Structures/Wallmounts/Signs/Posters/contraband.yml:
```diff
@@ -0,0 +1,7 @@
+- type: entity
+  parent: PosterBase
+  id: BaseCarpPosterContra
+  abstract: true
```
Isn't current convention that this should be above the parent and id?

Atakku replied:
tapind afaik isnt it? thats how i remember
but ye thats cuz i copied it from PosterBase and forgor to fix

Atakku committed:
conform to order conventions

Atakku committed:
Merge branch 'dev' into pr-more-posters

Atakku committed:
Merge remote-tracking branch 'upstream/dev' into pr-more-posters

Atakku committed:
Merge remote-tracking branch 'upstream/dev' into pr-more-posters

Atakku committed:
Merge remote-tracking branch 'upstream/dev' into pr-more-posters

Atakku committed:
Added some posters, refactored a bit

Atakku committed:
shut up the tests

Atakku commented:
Added 8 posters from [Nyanotrasen/Nyanotrasen#35b9f05ee963d0e9575229576c8bacc2d187d61b](https://github.com/Nyanotrasen/Nyanotrasen/commit/35b9f05ee963d0e9575229576c8bacc2d187d61b)

<img width="1033" height="358" alt="2026-03-14-044703_hyprshot" src="https://github.com/user-attachments/assets/689f44a8-07dd-4311-83d7-677582b0045e" />

I had to refactor and split up posters based on author because of differences in licensing

Atakku committed:
forgor

Atakku committed:
Refactor again, add new poster chicken_dinner, move donk_explosion to contraband posters

Atakku committed:
New poster: medicate

Atakku committed:
Refactored yet again

Atakku committed:
Calendar and shoeless posters from SL

Atakku committed:
Fixed a mistake + added two coords cancer posters

Atakku committed:
Added eye chart

Atakku commented:
Some of these posters have pretty lackluster copyright notices (blame SL), i will go over later and clean them up (resolve direct references)

Atakku committed:
self poster

Atakku committed:
keep it clean

Atakku commented:
also gotta change the no shoes poster reference

Atakku committed:
14 loko advert

Atakku committed:
Added dejavu poster + fixed licenses (oops)

Atakku committed:
Cleanup copyrights

Atakku committed:
More cleanup

Atakku commented:
is ready

Atakku commented:
misinput

Atakku commented:
<img src="https://media1.tenor.com/m/2Ksp9lFxlcUAAAAd/moistcritikal-penguinz0.gif" />

Atakku committed:
Forgor to save file

imatsoup reviewed (approved):
posters are cool
